webapp/src/main/webapp/themes/tenderfoot/templates/body/partials/individual/individual-visualizationFoafPerson.ftl

```diff
@@ -47,6 +47,8 @@
                             height = (chartHeight - margin.top - margin.bottom),
                             g = svg.append("g").attr("transform", "translate(" + margin.left + "," + margin.top + ")");
 
+                    svg.attr("width", chartWidth).attr("height",chartHeight)
```

Comment:
I assume that setting the `width` and `height` in lines 46 and 47 does not work?

Reply:
My guess is that those lines set the height and width of the visualization itself, but not the container it sits in.